app/frontend/src/components/admin/StatusTable.vue

```diff
@@ -62,8 +62,15 @@ export default {
     formatDate(date) {
       return date ? new Date(date).toLocaleString() : '';
     },
-    getLocalDateFromUtc(date){
-      return date ? new Date(date).toLocaleDateString() : '';
+    formatDateString(date){
+      if(date){
+        var [yyyy, mm, dd] = date.split('-');
```

Comment:
Can we just print the string as it comes back from the API? Having manual splits isn't maintainable.

Reply:
ok. it comes back in the format:
'2020-05-22'

Reply:
If you do want to format it, look at inspectionDateDisplay in InspectionPanel.vue

Reply:
Yeah either print it as is or use a formatter, no string manipulation, as that will possibly break if a format change from the API ever happens, FE should be able to handle it.